pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -110,6 +116,22 @@ func (p *perfCounterNodeStatsClient) startMonitoring() error {
 		p.collectMetricsData(cpuCounter, memWorkingSetCounter, memCommittedBytesCounter, networkAdapterCounter)
 	}, perfCounterUpdatePeriod)
 
+	// Cache the CPU usage every defaultCachePeriod
+	go wait.Forever(func() {
+		cpuValue, err := cpuCounter.getData()
```

Comment:
@feiskyer do you know how much time/resource it takes to get the data from the counter?

If it's bad, an alternative would be to cache a copy of `nodeMetrics` and compute the difference. WDYT?

Reply:
not sure, @PatrickLang have you noticed the time for this?